ui/raidboss/timeline.js

```diff
@@ -631,6 +640,11 @@ class TimelineUI {
     bar.toward = 'right';
     bar.style = !channeling ? 'fill' : 'empty';
 
+    if (e.style) {
```

Comment:
This is where you should handle new properties in the future, with ofc the correct setup in `resources/timerbar.js`

Reply:
I was wondering if it was possible to just set an array of styles here and pass that through all the way to timerbar.js and apply all of them there.  That way anybody could override whatever css style they wanted.  Do you think that would work?

Reply:
Should work like that now